src/main/java/kr/codesqaud/cafe/repository/users/DBUserRepository.java

```diff
@@ -1,33 +1,54 @@
 package kr.codesqaud.cafe.repository.users;
 
 import kr.codesqaud.cafe.domain.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Primary;
 import org.springframework.jdbc.core.BeanPropertyRowMapper;
 import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.stereotype.Repository;
 
 import javax.sql.DataSource;
-import java.time.LocalDateTime;
 import java.util.List;
 
 @Primary
 @Repository
 public class DBUserRepository implements UserRepository {
 
+    private Logger log = LoggerFactory.getLogger(getClass());
     private final JdbcTemplate jdbcTemplate;
 
     @Autowired
     public DBUserRepository(DataSource dataSource) {
         this.jdbcTemplate = new JdbcTemplate(dataSource);
     }
 
+    @Override
+    public User findDBUser(String userId, String password) {
```

Comment:
쿼리의 이름과 행동이 다르네요. 그리고 인터페이스랑 메소드 이름이 달라진 거 아닐까요? 
```suggestion
    public User findUserWithMatchedPasswrod(String userId, String password) {
```
정도의 네이밍이 어떨까 싶습니다.